docs/audit-log-filter/audit-log-filter-options.md

```diff
@@ -0,0 +1,419 @@
+
+A JSON log pruning operation is based on the values of [`audit_log_filter_rotate_on_size`](#audit_log_filter_rotate_on_size), [`audit_log_filter_max_size`](#audit_log_filter_max_size) and [`audit_log_filter_prune_seconds`](#audit_log_filter_prune_seconds). Setting only one of these values does not cause pruning.
+
+### audit_log_filter_read_buffer_size
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-read-buffer-size |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Bytes  |
+| Default | 32768  |
+
+This option is only supported for JSON-format files.
+
+The size of the buffer for reading from the audit log filter file. The `audit_log_filter_read()` reads only from this buffer size.
+
+| Values | Size |
+| --- | --- |
+| Default value | 32768 |
+| Minimum value | 32768 |
+| Maximum value | 4194304 |
+
+### audit_log_filter_rotate_on_size
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-rotate-on-size |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer |
+| Default | 0  |
+
+Performs an automatic log file rotation based on the size. The default value is 0 (zero).
+
+If the value is 0 (zero), the plugin does not automatically rotate the log file. If you set the value to less than 4096 resets the value to 0, and does not automatically rotate the log files. You can rotate the log files manually.
+
+If the value is greater than 0, when the log file size exceeds the value, the plugin renames the current file, and opens a new log file using the original name.
+
+If the value is not a multiple of 4096, the plugin truncates the value to the nearest multiple.
+
+### audit_log_filter_strategy
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-strategy |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | ASYNCHRONOUS  |
+
+Defines the Audit Log filter plugin's logging method. The valid values are the following:
+
+| Values | Description |
+| --- | --- |
+| ASYNCHRONOUS | Waits until there is outer buffer space |
+| PERFORMANCE | If the outer buffer does not have enough space, drops requests |
+| SEMISYNCHRONOUS | Operating system permits caching |
+| SYNCHRONOUS | Each request calls `sync()` |
+
+### Audit_log_filter_syslog_facility
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-syslog-facility |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | String  |
+| Default | LOG_USER |
+
+Specifies the syslog `facility` value. The option has the same meaning as the appropriate parameter described in the [syslog(3) manual](https://linux.die.net/man/3/syslog).
+
+### Audit_log_filter_syslog_ident
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-syslog-ident |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | String  |
+| Default | percona-audit  |
```

Comment:
percona-audit -> percona-audit-event-filter

Reply:
Which variable is this?